### PR TITLE
icalbuddy: add page

### PR DESCRIPTION
**Pull request by caiobep**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/osx/icalbuddy.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`icalbuddy -n -f eventsToday`
+
+- Show uncompleted tasks:
+
+`icalBuddy -f uncompletedTasks`
+
+- Show tasks for the next 3 days:
+
+`icalbuddy -n -f tasksDueBefore:today+3`
```
Comment (zdroid) [inline pages/osx/icalbuddy.md:16]:
```suggestion
`icalBuddy -n -f tasksDueBefore:today+3`
```

File: pages/osx/icalbuddy.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# icalBuddy
+
+> Command-line utility for prinitng events and tasks from the macOS calendar database.
+> More information: <https://hasseg.org/icalBuddy/>.
+
+- Show events later today:
+
+`icalbuddy -n -f eventsToday`
```
Comment (zdroid) [inline pages/osx/icalbuddy.md:8]:
```suggestion
`icalBuddy -n -f eventsToday`
```

File: pages/osx/icalbuddy.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# icalBuddy
+
+> Command-line utility for prinitng events and tasks from the macOS calendar database.
```
Comment (agnivade) [inline pages/osx/icalbuddy.md:3]: typo at printing.

File: pages/osx/icalbuddy.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`icalBuddy -n -f eventsToday`
+
+- Show uncompleted tasks:
+
+`icalBuddy -f uncompletedTasks`
+
+- Show tasks for the next 3 days:
+
+`icalBuddy -n -f tasksDueBefore:today+3`
```
Comment (agnivade) [inline pages/osx/icalbuddy.md:16]: today+3 should be in tokens (`{{}}`)

File: pages/osx/icalbuddy.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Show events in a time range:
+
+`icalBuddy -f eventsFrom:'{{start date}}' to:'{{end date}}'`
+
+- Show custom event list:
+
+`icalBuddy -f -npn -nc -iep "title,datetime" -ps "| : |" -po "datetime,title" -tf "" -df "%RD" -eed eventsToday+10`
```
Comment (agnivade) [inline pages/osx/icalbuddy.md:24]: IMO, this is a bit too complicated. There are far too many options which are not explained in the description at all, at which point this has just become a snippet to be copy-pasted.

**Comment by caiobep** [comment]
My apologies but I'm new to Open Source. What would be the best way to address those issues? Should I create another pull request?

**Comment by agnivade** [comment]
Yes

**Comment by sbrl** [comment]
Hey, welcome then @caiobep :-)

Yeah, a new pull request would be great. The GitHub web interface is great for this. You can navigate to the file in the main tldr-pages/tldr GitHub repository, and edit the file from there - and the GH web interface should guide you through the process  of committing the edit & opening a new pull request.

Just let us know if you need any assistance :-)

**Comment by caiobep** [comment]
Thanks for the help guys!! I’ve opened a new PR(#4094) addressing those issues, please let me know your opinions!

**Comment by sbrl** [comment]
Merged that

**Pull request merged**